File: matchteam/src/main/java/com/example/matchteam/dto/user/UserInfoDto.java
package com.example.matchteam.dto.user;

import lombok.Builder;
import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class UserInfoDto {
    private String email;
    private String name;
    private String department;
    private int studentNum;

    @Builder
    public UserInfoDto(String email, String name, String department, int studentNum) {
        this.email = email;
        this.name = name;
        this.department = department;
        this.studentNum = studentNum;
    }
}

File: MatchingTeam/app/src/main/java/com/example/matchingteam/dto/user/UserInfoDto.kt
package com.example.matchingteam.dto.user

import com.google.gson.annotations.SerializedName

data class UserInfoDto(
    @SerializedName("email") val email: String?,
    @SerializedName("name") val name: String?,
    @SerializedName("department") val department: String?,
    @SerializedName("studentNum") val studentNum: Int
)
